src/harness/loggedIO.ts

```diff
@@ -135,6 +146,88 @@ module Playback {
         wrapper.startRecord = (fileNameBase) => {
             recordLogFileNameBase = fileNameBase;
             recordLog = createEmptyLog();
+
+            if (underlying.args !== undefined && typeof underlying.args !== "function") {

```

Comment:
You can just do `typeof underlying.args !== "function"` since if it's undefined, you'll get `"undefined"`.